backend/experiment/actions/final.py

```diff
@@ -82,12 +218,17 @@ def action(self):
             "logo": self.logo,
         }
 
-    def get_social_media_config(self, session: Session) -> dict:
+    def get_social_media_config(self, session: Session) -> SocialMediaConfigDto | None:
```

Comment:
Again, this is a function that's *internally* called after initializing this action, and might actually confuse researcher-users of the infrastructure.